setup.py

```diff
@@ -33,13 +33,14 @@
     "pycodestyle >= 2.5.0, < 2.6.0",
     "pyflakes >= 2.1.0, < 2.2.0",
     "pyserial==3.4",
-    "qtconsole==4.4.3",
+    "qtconsole==4.6.0",
     "pgzero==1.2",
     "appdirs>=1.4.3",
     "semver>=2.8.0",
     "nudatus>=0.0.3",
     'black>=18.9b0;python_version > "3.5"',
     "Flask==1.0.2",
+    'pywin32>=227;platform_system == "Windows"',
```

Comment:
AFAICT, this is the only non-direct dependency that is listed here: is it needed, given that it is being explicitly brought in during the packaging process in `win_installer.py`?

If so, could we include a comment on why that is the case (much along the lines of the comment on `flake8` that is not a direct dependency, but `pycodestyle` and `pyflakes`, in which it depends, are).